GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -381,17 +381,22 @@ private void btnChooseFromRevision_Click(object sender, EventArgs e)
             bool previousValueBranchFilterEnabled = AppSettings.BranchFilterEnabled;
             bool previousValueShowCurrentBranchOnly = AppSettings.ShowCurrentBranchOnly;
             bool previousValueShowReflogReferences = AppSettings.ShowReflogReferences;
+            bool previousValueShowStashes = AppSettings.ShowStashes;
 
             try
             {
-                using FormChooseCommit chooseForm = new(UICommands, txtFrom.Text, showCurrentBranchOnly: true);
+                AppSettings.ShowStashes = false;
+                ObjectId firstParent = UICommands.GitModule.RevParse(Currentbranch.Text + "~");
```

Comment:
Does this also work for detached head?

Reply:
yes, failure to parse will give null (no error popup)

Reply:
I have a little changed that to improve when in detached head and have the same behavior (even if I really don't see the case of rebasing in detached head)